lib/web_console/templates/console.js.erb

```diff
@@ -1,25 +1,27 @@
-// DOM helpers
-function hasClass(el, className) {
+// A collection of helper functions for DOM elements
+var DOMHelpers = {};
+
+DOMHelpers.hasClass = function(el, className) {
   var regex = new RegExp('(?:^|\\s)' + className + '(?!\\S)', 'g');
   return el.className.match(regex);
-}
+};
 
-function addClass(el, className) {
+DOMHelpers.addClass = function(el, className) {
```

Comment:
You can just inline those in the object definition. E.g.

```javascript
var DOMHelpers = {
  addClass: function() {
    // ...
  }
}
```

IMO, we can even skip this step, and leave the helpers as is. They are wrapped in a closure by the templating system and act like private methods. Maybe we can just move them to the end of the file, so we see the more important code first.